roman.rb

```diff
@@ -0,0 +1,30 @@
+class Fixnum
+
+  def roman
+    {
+      1000 => "M",  
+      900  => "CM",  
+      500  => "D",  
+      400  => "CD",
+      100  => "C",  
+      90   => "XC",  
+      50   => "L",  
+      40   => "XL",  
+      10   => "X",  
+      9    => "IX",  
+      5    => "V",  
+      4    => "IV",  
+      1    => "I"  
+    }
+  end
+
+  def to_roman
+    n = self
```

Comment:
would benefit to leave comment that n is an instance of Fixnum, which gets converted to roman via to_roman